zmsapi/src/Zmsapi/Helper/ErrorHandler.php

```diff
@@ -62,12 +80,30 @@ public function __invoke(
             $status = $exception->getcode();
         }
         if ($exception->getCode() >= 500 || !$exception->getCode()) {
-            \App::$log->critical(
-                "[API] Fatal Exception: "
-                . " in " . $exception->getFile() . " +" . $exception->getLine()
-                . " -> " . $exception->getMessage()
-                . " | Trace: " . preg_replace("#(\s)+#", ' ', str_replace('\\', ':', $message->meta->trace))
-            );
+            if (
+                $exception instanceof \PDOException &&
+                stripos($exception->getMessage(), 'SQLSTATE') !== false &&
+                (stripos($exception->getMessage(), 'Connection refused') !== false ||
+                 stripos($exception->getMessage(), 'Connection timed out') !== false ||
+                 stripos($exception->getMessage(), 'Access denied') !== false)
+            ) {
+                $errorType = 'Connection refused';
+                if (stripos($exception->getMessage(), 'Connection timed out') !== false) {
+                    $errorType = 'Connection timed out';
+                } elseif (stripos($exception->getMessage(), 'Access denied') !== false) {
+                    $errorType = 'Access denied';
+                }
+                \App::$log->critical(
```

Comment:
Zu viel und dupliziert.

- Kann man nicht einfach wo "SQLSTATE" gibt einfach von trace \App::DB_PASSWORD ersetzen?
- In zmsslim helper methode packen?